.dockerignore

```
@@ -0,0 +1,14 @@
+# Git and system files
+.git
+.DS_Store
+
+# Local dependencies and build artifacts
+node_modules
+dist
+coverage
+
+# Environment files and logs
+.env
+.env.*
```

Pode ser simplificado como no `.gitignore`
```suggestion
.env*
```

---

De acordo.